src/guide-dist.md

```diff
@@ -72,6 +72,7 @@ Lets go over the distributions by type:
     The [`Standard`] and [`Uniform`] distributions each support a selection of
     these types (up to 12-tuples and 32-element arrays).
     This includes the empty tuple `()` and array.
+    For arrays of size larger than 32, enable the `min_const_gen` feature.
```

Comment:
I don't like the wording. Better:

> When using Rustc ≥ 1.51, enable the `min_const_gen` feature to support arrays larger than 32 elements.

Reply:
Done. Should I use the same phrasing in the docs in the original PR?

Reply:
I guess. There's also the thing I mentioned about non-equivalence with `Fill`; maybe I'll have a look at that if you like. Currently there's a mention on `Rng::gen` and some doc on `Standard`; probably there should be some on `Fill`.

Reply:
> There's also the thing I mentioned about non-equivalence with `Fill`; maybe I'll have a look at that if you like. Currently there's a mention on `Rng::gen` and some doc on `Standard`; probably there should be some on `Fill`.

You mentioned this can somehow be solved with `min_specialization`, which I am unfamiliar with. Did you mean we should solve this now or in the future?